### PR TITLE
Dutch Locale

### PR DESCRIPTION
**Pull request by Jespiee**

### CODE REVIEW

File: src/_locales/nl.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  "label-show-lyrics": "Bekijk songtekst (Beta)",
+  "label-unknown-artist": "Onbekende Artiest",
+  "label-unknown-song": "Onbekend nummer",
+  "label-version": "Versie",
+
+  "lastfm-login-authorized": "Ingelogd",
+  "lastfm-login-authorizing": "Inloggen",
+  "lastfm-login-button-text": "Log in op Last.FM",
+  "lastfm-login-error": "Je hebt het inloggen geannuleerd, probeer het opnieuw",
+  "lastfm-login-not-authorized": "Niet Ingelofd",
```
Comment (Azalthur) [inline src/_locales/nl.json]: 31 should be: 
`"lastfm-login-not-authorized": "Niet Ingelogd"`

File: src/_locales/nl.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  "modal-confirmUninstall-content": "We hebben een oudere versie van GPMDP gedetecteert op dit systeem. Dit is heel normaal als je net geupdate hebt. Om de grootte van dit programma een beetje te beperken gaan we de oude versie van GPMDP verwijderen. Hiervoor zijn administrator rechten vereist.",
+  "modal-goToURL-title": "Vul hier de Google Play Music URL in waarnaar u wilt navigeren:",
+  "modal-welcome-title": "Welkom bij versie",
+  "modal-welcome-content": "We hopen dat u kunt genieten van de nieuwe update!",
+
+  "playback-label-next-track": "Volgende nummer",
+  "playback-label-play-pause": "Play / Pause",
+  "playback-label-previous-track": "Vorige Nummer",
+  "playback-label-stop": "Stop",
+  "playback-label-thumbs-down": "Duimp omlaag",
```
Comment (Azalthur) [inline src/_locales/nl.json]: 57 should be  `"playback-label-thumbs-down": "Duim omlaag",`

File: src/_locales/nl.json [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  "button-text-not-now": "Niet nu",
+
+  "label-about": "Over",
+  "label-desktop-settings": "Instellingen",
+  "label-donate": "Donatie",
+  "label-help": "Help",
+  "label-issues": "Problemen",
+  "label-learn-more": "Lees Meer",
+  "label-loading-devices": "Apparaten laden...",
+  "label-pause-after-song": "Pause after this song",
```
Comment (Kriskras99) [inline src/_locales/nl.json]: Should be
`"label-pause-after-song": "Stop na dit lied",`

File: src/_locales/nl.json [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+  "settings-option-mini-always-show": "Altijd nummerinformatie laten zien",
+  "settings-option-mini-ontop": "Altijd op top",
+
+  "title-color-picker": "Color Picker Magic",
+  "title-settings": "Instellingen",
+  "title-settings-audio": "Geluid",
+  "title-settings-general": "Algemeen",
+  "title-settings-hotkeys": "Hotkeys",
+  "title-settings-lastfm": "LastFM",
+  "title-settings-mini": "Minispelerr",
```
Comment (Kriskras99) [inline src/_locales/nl.json]: Should be
`"title-settings-mini": "Minispeler",`

**Comment by MarshallOfSound** [comment]
@Jespiee Can you please either update the PR or discuss with @Kriskras99 the suggested changes.  Looking to get these locales merged soon.

Also can you rebase onto latest master

**Comment by MarshallOfSound** [comment]
Ping @Jespiee @Kriskras99

**Comment by Jespiee** [comment]
All their changes are good, just some typing errors by me.. Thanks for checking it for me! 

Only at line 20 i'd say: "Pauzeer na dit nummer". It sounds a bit better.

**Comment by MarshallOfSound** [comment]
@Jespiee Also worth noting that when you update this, you need to rebase onto master.

There have been a few new strings added since this original PR

**Comment by Jespiee** [comment]
Alright, will do that tonight when I get home from school!

**Comment by MarshallOfSound** [comment]
@Jespiee Please rebase and ping when complete so I know

File: src/_locales/nl.json [hunk 5]
```diff
@@ -42,6 +42,7 @@
 
   "modal-confirmTray-title": "Kijk uit",
   "modal-confirmTray-content": "Het programma minimaliseert zich naar het systeemvak bij het afsluiten (wat u zonet probeerde te doen) van het programma. Je kan dit gedrag veranderen in de instellingen van GPMDP.",
+  "modal-confirmOpenPort-content": "De PlaybackAPI kon niet worden gestard omdat Windows Firewall dit blokkeert. Om dit op te lossen zijn administartor-machtigingen nodig.",
```
Comment (Kriskras99) [inline src/_locales/nl.json]: Shouldn't  "gestard" be "gestart" @Jespiee?

Comment (jessehouwing) [inline src/_locales/nl.json]: Yes

**Comment by MarshallOfSound** [comment]
Closing PR due to how out of date it is and lack of response

**Pull request closed without merging**